dev/ci/scripts/ci-metarocq.sh

```diff
@@ -5,12 +5,12 @@ set -e
 ci_dir="$(dirname "$0")"
 . "${ci_dir}/ci-common.sh"
 
-git_download metacoq
+git_download metarocq
 
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 export COQEXTRAFLAGS='-native-compiler no'
-( cd "${CI_BUILD_DIR}/metacoq"
+( cd "${CI_BUILD_DIR}/metarocq"
   [ -e pcuic/metacoq-config ] || ./configure.sh local
```

Comment:
NB: it seems this file is still named metacoq-config

Reply:
You should probably fix this downstream before merging this PR.

Reply:
why?

Reply:
Because this way we won't have to merge two PRs for the same change.

Reply:
I made the change in the 9.0 branch, renaming to metarocq-config. To merge in master I will need to make PRs on CI projects depending on metacoq. Are there any?

Reply:
stdlib, but AFAICT they use nix and don't try to optimize incremental builds by skipping configure if the file is there

Reply:
stdlib depending on metarocq? You mean in their reverse CI?

Reply:
yes
Why would projects depending on metarocq care about this config file?